### PR TITLE
[4675] Create publish history table and use that to report

### PR DESCRIPTION
**Pull request by dejan-brkic**
Updated url for deployment history

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/4675

### CODE REVIEW

File: static-assets/components/cstudio-common/common-api.js [hunk 1]
```diff
@@ -3018,7 +3018,7 @@ var nodeOpen = false,
 
       // DEPLOYMENT SERVICES
       // READ OPS
-      getDeploymentHistoryServiceUrl: '/api/1/services/api/1/deployment/get-deployment-history.json',
+      getDeploymentHistoryServiceUrl: '/api/2/publish/history.json',
```
Comment (rart) [inline static-assets/components/cstudio-common/common-api.js:3021]: This const is not used anymore. We can get rid of it.

File: ui/app/src/services/dashboard.ts [hunk 2]
```diff
@@ -109,5 +109,5 @@ export function fetchLegacyDeploymentHistory(
     ...(filterBy && { filterType: filterBy })
   });
 
-  return get(`/studio/api/1/services/api/1/deployment/get-deployment-history.json${qs}`).pipe(pluck('response'));
+  return get(`/studio/api/2/publish/history.json${qs}`).pipe(pluck('response'));
```
Comment (rart) [inline ui/app/src/services/dashboard.ts:112]: Is API 2 returning same response format?

**Comment by rart** [comment]
Was the old/api1 removed completely? Will calling it redirect or fail?

File: ui/app/src/services/dashboard.ts (hunk 2 above)
Comment (dejan-brkic) [reply ui/app/src/services/dashboard.ts:112]: it is responding in the same format as 3.1

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**